### DISCUSSION TITLE
Azure IPAM Support

### DISCUSSION THREAD

**Discussion by Darius4000** [Azure/ipam#38] [General]
Is there any support contact? If there is, what type and level of support are available for enterprise organizations? We are doing research to implement this utility for our Azure cloud, and management asked this question.

**Comment by DCMattyG** [comment]
Hi @Darius4000, thank you so much for reaching out to us! As of now, Azure IPAM is an open source platform, and as such is supported primarily through GitHub discussions and/or issues. We are working closely with the Azure product groups to bring an Azure-native solution to life in the near future, but for now we've created this project to fill the current gap. If you have any questions/comments/issues, please feel free to reach out here, but at this time there is no formal support for Azure IPAM. We'll make sure to respond as quickly as possible, and we're always looking for constructive feedback to make the platform better for everyone.
